test/fixtures/redis_cluster/main.tf

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "random" {
+  version = "~> 2.0"
+}
+
+resource "random_pet" "main" {
+  length    = 1
+  prefix    = "redis"
+  separator = "-"
+}
+
+resource "google_compute_network" "test" {
+  project                 = "${var.project_id}"
+  name                    = "${random_pet.main.id}"
+  auto_create_subnetworks = "false"
+}
+
+resource "google_compute_subnetwork" "test" {
+  project                  = "${var.project_id}"
+  region                   = "us-east1"
+  network                  = "${google_compute_network.test.self_link}"
+  name                     = "${random_pet.main.id}"
+  ip_cidr_range            = "10.0.0.0/24"
+  private_ip_google_access = "true"
+}
+
+module "example" {
+  source = "../../../examples/redis_cluster"
+
+  project_id        = var.project_id
+  bucket_name       = random_pet.main.id
+  client_ip_range   = "0.0.0.0/0"
+  cluster_name      = random_pet.main.id
+  region            = "us-east1"
+  zone_a            = "us-east1-a"
+  zone_b            = "us-east1-a"
+  zone_c            = "us-east1-a"
+  network_project   = var.project_id
+  network           = random_pet.main.id
```

Comment:
This needs to be a reference to the google_compute_network resource above so it gets passed all the way through.

Ex. `google_compute_network.test.name`

Likewise for subnet.